internal/providers/terraform/ibm/testdata/resource_instance_test/resource_instance_test.tf

```diff
@@ -43,27 +43,6 @@ resource "ibm_resource_instance" "resource_instance_power_iaas" {
   resource_group_id = "default"
 }
 
-resource "ibm_resource_instance" "resource_instance_appconnect_ent" {
-  name              = "appconnect-ent"
-  service           = "appconnect"
-  plan              = "appconnectplanenterprise"
-  location          = "us-south"
-}
-
-resource "ibm_resource_instance" "resource_instance_appconnect_pro" {
-  name              = "appconnect-pro"
-  service           = "appconnect"
-  plan              = "appconnectplanprofessional"
-  location          = "us-south"
-}
-
-resource "ibm_resource_instance" "resource_instance_appconnect_lite" {
-  name              = "appconnect-lite"
-  service           = "appconnect"
-  plan              = "lite"
-  location          = "us-south"
-}
-
```

Comment:
You probably would want to remove the corresponding usages in the `internal/providers/terraform/ibm/testdata/resource_instance_test/resource_instance_test.usage.yml` file

Reply:
thank you